controllers/grid/queries/QueryNotesGridHandler.inc.php

```diff
@@ -162,7 +162,9 @@ public function getRequestArgs()
      */
     public function loadData($request, $filter = null)
     {
-        return $this->getQuery()->getReplies(null, NoteDAO::NOTE_ORDER_DATE_CREATED, \PKP\db\DAO::SORT_DIRECTION_ASC, $this->getCanManage(null));
+        return $this->getQuery()
```

Comment:
I suspect this isn't working as expected -- it should be an associative array. Paging doesn't need to be considered here, but otherwise consider the example:

https://github.com/pkp/pkp-lib/blob/30c5c2df4eecb6e9eca4df320cf292e296874ad9/controllers/grid/settings/user/UserGridHandler.inc.php#L192..L198

Reply:
You are right. The notes where being displayed but actions like `delete` was not successful. Because of [that](https://github.com/pkp/pkp-lib/pull/8075#discussion_r940046361) change, the `getByAssoc`, and, thus the `getReplies` function returns an associative array.

It seems that it works as it supposed to - Should I follow the example you have provided though?

Reply:
Do you get a warning like this during rebase?
```
warning: inexact rename detection was skipped due to too many files.
```
If so, try (at least temporarily)...
```
git config merge.renamelimit 50000
```

Reply:
Thanks for the note. I didn't get something like that, but I will keep that in mind.